plugins/artnet/ArtNetPort.cpp

```diff
@@ -35,10 +35,15 @@ using ola::rdm::RDMCommand;
 using std::string;
 using std::vector;
 
+namespace {
+static const uint8_t ARTNET_UNIVERSE_COUNT = 16;
+};  // namespace
+
 void ArtNetInputPort::PostSetUniverse(Universe *old_universe,
                                       Universe *new_universe) {
   if (new_universe)
-    m_node->SetOutputPortUniverse(PortId(), new_universe->UniverseId() % 0xf);
```

Comment:
ArtNetNode uses (port->universe_address & 0xf) for stuff, we should probably switch to the same constant throughout and ideally use the same method of processing.